ff_derive/src/lib.rs

```diff
@@ -1267,12 +1267,12 @@ fn prime_field_impl(
             const ONE: Self = R;
 
             /// Computes a uniformly random element using rejection sampling.
-            fn random<R: ::ff::derive::rand_core::RngCore + ?Sized>(rng: &mut R) -> Self {
+            fn try_from_rng<R: ::ff::derive::rand_core::TryRngCore + ?Sized>(rng: &mut R) -> core::result::Result<Self, R::Error> {
```

Comment:
We always use `::crate_name` notation to defend against the import decisions within the module in which the derive is used (see the rest of `ff_derive` for examples).
```suggestion
            fn try_from_rng<R: ::ff::derive::rand_core::TryRngCore + ?Sized>(rng: &mut R) -> ::core::result::Result<Self, R::Error> {
```